extern/sector-storage/stores/remote.go

```diff
@@ -357,6 +357,67 @@ func (r *Remote) readRemote(ctx context.Context, url string, spt abi.RegisteredS
 	return resp.Body, nil
 }
 
+func (r *Remote) CheckAllocated(ctx context.Context, s storage.SectorRef, offset, size abi.PaddedPieceSize, ft storiface.SectorFileType) (bool, error) {
+	if ft != storiface.FTUnsealed {
+		return false, xerrors.Errorf("CheckAllocated only supports unsealed files")
+	}
+
+	paths, _, err := r.local.AcquireSector(ctx, s, ft, storiface.FTNone, storiface.PathStorage, storiface.AcquireMove)
+	if err != nil {
+		return false, xerrors.Errorf("acquire local: %w", err)
+	}
+
+	path := storiface.PathByType(paths, ft)
+	if path == "" {
```

Comment:
When is `path == ""` ? This function always returns and string based on the `ft`, no?